docs/api/internal.json

```diff
@@ -82,19 +82,38 @@
                             "example": false
                           },
                           "job_type":  {
-                            "type": "string",
+                            "type": "array",
                             "enum": [
                               "freelance",
                               "full-time",
                               "internship",
                               "part-time",
                               "temporary",
                               "other"
-                            ]
+                            ],
+                            "items" : {
+                              "type": "string",
+                              "example": "Full Time"
+                            }
                           },
                           "salary":  {
-                            "type": "string",
-                            "example": "$50,000 - $100,000"
+                            "type": "array",
```

Comment:
I think these 2 lines we need to remove, right?

```
"type": "array",
"items": {
```

Reply:
Fixed in c7dc9536e2935b9c52f42781fc9f23f99d60e14f

Reply:
I think now we removed an extra curly brace.
You can use this tool to validate the document: https://editor.swagger.io/ 😉

Reply:
Nice tool!  Fixed now in 0c48b308d226b71423baeee481f4a061d844ada5